### DISCUSSION TITLE
Unable to read data from the transport connection: The connection was closed.

### DISCUSSION THREAD

**Discussion by yogesh224** [postalserver/postal#2590] [Help with using Postal]
Hello,
After installing postal, when I try to send email, I get error
" SMTP send error
Failure sending mail.
Unable to read data from the transport connection: The connection was closed."

Can someone point what could be the issue.

Regards

**Comment by willpower232** [comment]
You could try having a look at the logs of the SMTP server to check for any errors that occurred.
